src/pages/docs/guides/chat/build-livestream.mdx

```diff
@@ -0,0 +1,210 @@
+
+Be aware of the length of the interval you configure, over which messages are batched. If it is too high, then users may notice the delay from messages being sent, however this is normally less of a concern for fast-paced livestreams. An interval of 1-2 seconds is likely sufficient for most livestream chat applications.
+
+#### Message ordering
+
+Messages will be received by users in the order that they are broadcast, by the region that the user is connected to. This may differ slightly to users in other regions, which for a fast-paced livestream is usually acceptable.
+
+If you want to ensure that every user sees every message in exactly the same order, you can use the [global ordering of messages](/docs/chat/rooms/messages#global-ordering) to do this.
+
+<Code>
+```javascript
+sortedMessages.sort((a, b) => (a.before(b) ? -1 : 1));
+```
+</Code>
+
+### User counts
+
+Most livestream applications display the number of users currently participating. There are two different places that you'll want to show this:
+
+* Within each stream so that users can see up-to-date totals of those in the stream.
+* If you are hosting multiple streams then you will want to show a count against each one to display the most popular when users are choosing which to join.
+
+#### Within a stream
+
+Use the [occupancy](/docs/chat/rooms/occupancy) feature to show the count of users within a stream. You can subscribe each user to the occupancy feature and use the `connections` metric to display the number of users attached to the room.
+
+#### On a stream list
+
+If your application hosts multiple streams, then you likely want to display the number of users watching each one. Use a server to periodically retrieve [occupancy](/docs/chat/rooms/occupancy#retrieve) from all streams via a REST call to display this information on a stream list.
+
+### Send reactions to the room
+
+[Room reactions](/docs/chat/rooms/reactions) are ephemeral reactions that users can send to the room, such as 👍or ♥️. They are not persisted and are used for showing sentiment for what is happening in the stream.
+
+Ably recommends you enable server-side batching if you use room reactions in your stream. This is because the highest volumes of reactions are normally sent in a short space of time, such as when a team scores in a game. Server-side batching will reduce the costs associated with the reactions occurring over that short time period.
+
+### Moderate your stream
+
+Keeping your chat stream free of abuse and toxic behavior is paramount to providing a safe and enjoyable experience for your users.
+
+Use AI models to vet content in realtime and keep your stream free of abusive content. Ably recommends using an 'after-publish' form of moderation for livestream chat. This is where messages are briefly visible to users, and then removed after moderation has been run against them.
```

Comment:
```suggestion
Use AI models to vet content in realtime and keep your stream free of abusive content. Ably recommends using an 'after-publish' form of moderation for livestream chat. This is where messages are briefly visible to users, and then removed after moderation has been run against them. The reason for this is to avoid the latency penalty of moderating inline: in busy rooms, users likely will not have time to comprehend a message before moderation removes it.
```

(Or some better words to that effect!)